src/components/book/book.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+
+const Book = ({ title, categorie }) => (
+  <tr>
+    <th>{title}</th>
+    <th>{categorie}</th>
+  </tr>
+);
```

Comment:
## Implement the missing book id section 
- [ ] Our `book` component should represent a single row of books with `id`, `title` and `category` [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Book/RepresentARowInBooksTable.png). We are currently missing the `id` part, lets add it.